CHANGES.rst

```diff
@@ -655,6 +655,9 @@ Bug Fixes
   - Fix convenience functions (``getdata``, ``getheader``, ``append``,
     ``update``) to close files. [#4786]
 
+  - Removed raising of AssertionError that could occur after closing or
```

Comment:
This is under 1.1.3 as per original PR. But I can't find that milestone tag anymore. Should I move this to 1.2?